shared/src/commonTest/kotlin/by/audiobooks/mob/model/source/DataDeserializationTest.kt

```diff
@@ -7,12 +7,16 @@ class DataDeserializationTest {
 
     @Test
     fun deserializationTest() {
+
         val data = datasourceDeserializer.decodeFromString<Data>(DataJsonExample.data)
-        assertTrue(data.books.size == 3)
-        assertTrue(data.people.size == 3)
-        assertTrue(data.tags.size == 3)
-        assertTrue(data.linkTypes.size == 3)
-        assertTrue(data.publishers.size == 3)
+
+        assertTrue {
+            data.books.size == 3 &&
+            data.people.size == 3 &&
+            data.tags.size == 3 &&
+            data.linkTypes.size == 3 &&
+            data.publishers.size == 3
+        }
```

Comment:
Why this change? Having individual asserts is easier for debugging.

Reply:
Actually it does not any matter, and any single assertion will be more that enough since deserialization will fail if json payload will have an inexactness. 
I can delete redundant verifications if you like.

Reply:
I see. If the the main goal of the test to verify that it correctly parses JSON then doing all those checks is reduntant indeed. Probably keep just 1 a sanity check.

Reply:
fixed